packages/blade/src/components/Drawer/Drawer.web.tsx

```diff
@@ -133,7 +130,12 @@ const _Drawer = ({
               ) : null}
               <AnimatedDrawerContainer
                 className={isVisible ? SHOW_DRAWER : ''}
-                width={{ base: '90%', s: '375px', m: '420px' }}
+                isFirstDrawerInStack={isFirstDrawerInStack}
+                width={{
+                  base: stackingLevel > 1 ? 'calc(90% - 16px)' : '90%',
```

Comment:
Is this for mobile? 

Let's properly test multiple drawers to see if we are not missing any edge case

Reply:
changed the approach so resolving this